BrainPortal/app/views/data_providers/new_normal.html.erb

```diff
@@ -0,0 +1,154 @@
+
+<%-
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2012
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+-%>
+
+<% title 'Add New Data Provider' %>
+
+<h2>Add New SSH based Data Provider</h2>
+
+<%= error_messages_for(@provider) %>
+
+<%= form_for @provider, :as => :data_provider, :url => {:action => "create_for_normal_user"}, :datatype => "script" do |f| -%>
+
+  <div class="generalbox">
+
+    <span title="The name of the data provider.">
```

Comment:
Since this form is meant for normal users instead of sysadmins, replace all the "title" in the spans with field_explanations.

Also, the original HTML was wrong, and all these SPANs are supposed to be DIVs.

Reply:
ok, maybe `show table` then